### ISSUE TITLE
Different marker behaviour across browsers

### ISSUE THREAD

**Issue by lolcookie** [FormidableLabs/victory#1406]
# Bugs and Questions

### Checklist

- [X] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [X] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [X] I am using the latest version of Victory

- [X] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

I have a compass component that I've implemented using markers, but compass only points the right way on chrome, on firefox and edge it doesn't change direction. 

### Reproduction
https://codesandbox.io/s/modest-gauss-und1s

**Comment by boygirl** [comment]
@lolcookie that's very strange! I took a look at your sandbox in chrome and firefox and I see what you mean about firefox being offset by 90 degrees. I changed the styling of the curve to make it more apparent:

Firefox:
<img width="705" alt="Screen Shot 2019-10-15 at 2 47 55 PM" src="https://user-images.githubusercontent.com/3719995/66872755-92795b80-ef5b-11e9-8788-438b8790d27f.png">

Chrome:
<img width="691" alt="Screen Shot 2019-10-15 at 2 48 47 PM" src="https://user-images.githubusercontent.com/3719995/66872771-9a390000-ef5b-11e9-941b-1a48d2fba990.png">

 Unfortunately I don't think this is an issue that can be solved in Victory, since there's nothing specific to markers in the lib. 
Since it looks like you're comfortable writing your own paths in the SVG mini language, you might try implementing this using `VictoryScatter` instead, and writing a custom `getPath` function for your `Point` component (in order to make it ship shaped) and then maybe rotating the whole rendered path component by the correct angle.
